website/content/docs/concepts/namespace-api-lock.mdx

```diff
@@ -0,0 +1,84 @@
+---
+layout: docs
+page_title: Namespace API Lock
+description: Lock the Vault API on a per-namespace basis.
+---
+
+# Lock/Unlock
+
+Vault makes the API available (unlocked) by default for all namespaces. In
+this state, Vault can respond to all API/CLI ('API' from here on out) requests
+as it usually would.
+
+A Vault Administrator can lock the API for particular namespaces. In this state,
+Vault will block all but a select few API endpoints from responding to clients
+operating in a locked namespace (or a descendant of a locked namespace). The
+endpoints that do respond, the exempt paths, are largely the same as the Vault
+unauthenticated paths. They are mainly used for checking the status of various
+systems (e.g. `sys/health`), or for unlocking the API.
+
+When the API is locked for a particular namespace, it is also locked for all
+descendants of that namespace. If the API was already locked for a descendant
+that lock will remain, but Vault must be unlocked for the parent before
+unlocking the descendant.
+
+## Why?
+
+Blocking access to much of Vault can be an important break-glass tool in the
+event of unexpected behavior.
+
+For HCP Vault, this provides functionality analogous to sealing Vault, without
```

Comment:
This transit to HCP Vault here is a bit confusing to me. 

Unless you are trying to make that analogy between seal/unseal and namespace for Vault. If that is the intent, then I would just start out by saying: The namespace lock/unlock functionality is analogous to the seal/unseal feature, but without requiring the Vault administrator to request the Managed Service Provider to seal/unseal Vault.

Something along those lines?